src/components/TopicsList/TopicCard/TopicCard.styles.js

```diff
@@ -36,6 +36,11 @@ export const useStyles = makeStyles(theme => ({
     accordion: {
         cursor: 'default !important'
     },
+    activeAccordion: {
+        cursor: 'default !important',
+        backgroundColor: '#f2e6ff',
```

Comment:
I'd suggest this colour: #edf0ff - current one kinda doesn't fit with our colour scheme for me:? 😕